### ISSUE TITLE
Yarn 0.24.6 not available in debian repository

### ISSUE THREAD

**Issue by scinos** [yarnpkg/yarn#3541]
**Do you want to request a *feature* or report a *bug*?**

Report a bug

**What is the current behavior?**

Running

    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
    echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
    sudo apt-get update && sudo apt-get install yarn

Installs yarn 0.24.5 despite 0.24.6 being released a week ago.

**What is the expected behavior?**

Yarn 0.24.6 is installed

**Please mention your node.js, yarn and operating system version.**

Ubuntu 16.04.2 LTS

**Comment by welkie** [comment]
Can confirm this. I just tried to install yarn on my fresh Kubuntu 17.10 install, and it was able to install it, but it grabbed version 0.24.5. I saw a warning about using an outdated version when I began to use it.

**Comment by tbroyer** [comment]
Moreover, Yarn warns:
```
warning Your current version of Yarn is out of date. The latest version is "0.24.6" while you're on "0.24.5".
```

**Comment by brendo** [comment]
The same is true for [NPM](https://www.npmjs.com/package/yarn) as well, the last version published there is 0.24.5.

**Comment by Daniel15** [comment]
Good catch. The RPM and Debian packages are updated as part of the same build job, and it seems like the RPM one isn't working properly:
```
yarn-0.24.6-1.noarch.rpm:
send: spawn id exp3 not open
    while executing
"send -- "blank\r""
    (file "./sign-rpm.sh" line 6)

```

**Comment by Daniel15** [comment]
Seems like a new release of RPM broke the script, as it fixed [Debian bug #858998](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=858998)
```
rpm (4.12.0.2+dfsg1-2) unstable; urgency=medium

  * Adjust debian/watch to new upstream website.
  * Add patch to improve rpmsign behavior with gpgp2 (Closes: #858998).

 -- Michal ÄŒihaÅ™ <nijel@debian.org>  Wed, 29 Mar 2017 15:24:58 +0200
```

Fixed in https://github.com/yarnpkg/releases/commit/0e638627e97f7643a76abe294aca719d0e20bf52, and then the update script kicked in and worked as expected: https://github.com/yarnpkg/releases/commit/e9b29323d6b782c568e0720874093c4f49e70875
